### PR TITLE
[163] created Delete-Event

### PR DESCRIPTION
**Pull request by RockTF**
"Merge is required after task https://github.com/OlhenShu/ProjectStage_Team1_May/issues/157." 
PR - https://github.com/GreenCityProject/GreenCityMVP21_team1_May/pull/64

### CODE REVIEW

**Review by ushh789 — APPROVED** [review]

File: service/src/main/java/greencity/service/EventServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+    /**
+     * Deletes an event with the specified eventId and name.
+     *
+     * @param eventId The ID of the event to be deleted.
+     * @param email The email of the user who is deleting the event.
+     */
+    @Override
+    public void delete(Long eventId, String email) {
+        UserVO userVO = restClient.findByEmail(email);
```
Comment (BerserkrMM) [inline service/src/main/java/greencity/service/EventServiceImpl.java]: You can use userService.findByEmail() instead sending http request to User Service

**Comment by BerserkrMM** [comment]
add `"/event/delete/{eventId}",` to security config

**Pull request closed without merging**